furms-domain/src/main/java/io/imunity/furms/domain/authz/roles/ResourceId.java

```diff
@@ -3,7 +3,7 @@
  * See LICENSE file for licensing information.
  */
 
-package io.imunity.furms.core.config.security.user.resource;
```

Comment:
As this is effect of my comment to CR: authz is fine but I don't think we need .roles at the end. In this package you have also non-role related stuff as this class.